### PR TITLE
Docs: Add aiohttp-pydantic to third party libraries

### PR DESCRIPTION
**Pull request by Maillol**
## What do these changes do?

Add aiohttp-pydantic to third party libraries in the documentation.

## Checklist

- [ ] I think the code is well written
- [ ] Unit tests for the changes exist
- [X] Documentation reflects the changes
- [ ] If you provide code modification, please add yourself to `CONTRIBUTORS.txt`
  * The format is &lt;Name&gt; &lt;Surname&gt;.
  * Please keep alphabetical order, the file is sorted by names.
- [ ] Add a new news fragment into the `CHANGES` folder
  * name it `<issue_id>.<type>` for example (588.bugfix)
  * if you don't have an `issue_id` change it to the pr id after creating the pr
  * ensure type is one of the following:
    * `.feature`: Signifying a new feature.
    * `.bugfix`: Signifying a bug fix.
    * `.doc`: Signifying a documentation improvement.
    * `.removal`: Signifying a deprecation or removal of public API.
    * `.misc`: A ticket has been closed, but it is not of interest to users.
  * Make sure to use full sentences with correct case and punctuation, for example: "Fix issue with non-ascii contents in doctest text files."

### CODE REVIEW

File: docs/third_party.rst [hunk 1]
```diff
@@ -139,6 +139,9 @@ period ask to raise the status.
 - `aiohttp-validate <https://github.com/dchaplinsky/aiohttp_validate>`_
   Simple library that helps you validate your API endpoints requests/responses with json schema.
 
+- `aiohttp-pydantic <https://github.com/Maillol/aiohttp-pydantic>`_
+  An aiohttp.View to validate request body, qs and headers regarding function annotations and generate Open API doc. Python 3.8+ required.
```
Comment (webknjaz) [inline docs/third_party.rst]: Looks like the spellchecker doesn't like `qs`. Could you please change it? We need CI to be green before merging.

Comment (Maillol) [reply docs/third_party.rst]: Sorry, I fixed that typo and added a change file in order to Timeline protection changes to green.

Comment (Maillol) [reply docs/third_party.rst]: Is SAP SE the owner of aiohttp? The CLA assistant ask me to sign your License Agreement with SAP Se.

Comment (asvetlov) [reply docs/third_party.rst]: My fault. The CLA text was fixed.
Now it is pretty close to the Python CLA.
The reference is here: https://gist.github.com/asvetlov/dbce6679052f5c89d3d13f501107c1d4

Sorry for misleading.

File: docs/third_party.rst [hunk 2]
```diff
@@ -139,6 +139,9 @@ period ask to raise the status.
 - `aiohttp-validate <https://github.com/dchaplinsky/aiohttp_validate>`_
   Simple library that helps you validate your API endpoints requests/responses with json schema.
 
+- `aiohttp-pydantic <https://github.com/Maillol/aiohttp-pydantic>`_
+  An aiohttp.View to validate request body, query string and headers regarding function annotations and generate Open API doc. Python 3.8+ required.
```
Comment (asvetlov) [inline docs/third_party.rst]:
```suggestion
  An ``aiohttp.View`` to validate the HTTP request's body, query-string, and headers regarding function annotations and generate Open API doc. Python 3.8+ required.
```

File: docs/third_party.rst (hunk 1 above)
Comment (Maillol) [reply docs/third_party.rst]: Ok great! Now the received link point to right text. Thanks

**Review by asvetlov — APPROVED** [review]

File: docs/third_party.rst (hunk 2 above)
Comment (asvetlov) [reply docs/third_party.rst]: Thanks!

**Pull request merged**